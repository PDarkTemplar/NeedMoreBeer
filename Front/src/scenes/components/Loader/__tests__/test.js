// @flow
import { shallow } from 'enzyme';

import React from 'react';

import { WrappedComponent as Loader } from '../index';

test('should render loading', () => {
    const loader = shallow(<Loader loading />);

    expect(loader).toMatchSnapshot();
});

test('should not render loading', () => {
    const loader = shallow(<Loader loading={false} />);

    expect(loader).toMatchSnapshot();
});
