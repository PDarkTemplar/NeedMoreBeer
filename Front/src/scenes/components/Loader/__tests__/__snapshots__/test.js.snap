// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not render loading 1`] = `
<Loader
  noAnimation={false}
  overlay={true}
  visible={false}
/>
`;

exports[`should render loading 1`] = `
<Loader
  noAnimation={false}
  overlay={true}
  visible={true}
/>
`;
